src/RemoteStorageDetails/RemoteStorageDetailsContainer.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import {
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import RemoteStorageDetails from './RemoteStorageDetails';
+
+const RemoteStorageDetailsContainer = ({
+  mutator,
+  match: { params },
+}) => {
+  const [storage, setStorage] = useState({});
+  const [isLoading, setIsLoading] = useState();
+
+  useEffect(() => {
+    setIsLoading(true);
+    mutator.configurations.GET({
+      path: `remote-storage/configurations/${params.id}`,
```

Comment:
this might return 404 response, maybe it makes sense to handle that case as well